INFO:albumentations.check_version:A new version of Albumentations is available: 1.4.11 (you have 1.4.10). Upgrade using: pip install --upgrade albumentations
Namespace(batch=64, dataset='datasets/rosbotxl_data', epochs=100, log_interval=50, lr=0.0001, noisevar=15, robustification=True)
Found data_cleaned.csv in datasets/rosbotxl_data/002
Finished intaking image paths!
Retrieving output distribution....
Moments of distribution: {'shape': (1624,), 'mean': -0.007825472621531614, 'median': 0.0, 'var': 0.028102853014347368, 'skew': -0.1196248711117521, 'kurtosis': 6.940936029081136, 'max': 0.9586232900619508, 'min': -1.0}
Total samples: 1624
time to load dataset: 0.6770901679992676
iteration='DAVE2v3-2560x720-lr0.0001-100epoch-64batch-lossMSE-1Ksamples-INDUSTRIALandHIROCHIandUTAH-noiseflipblur'
device=device(type='cuda')
Processed 18 transformations for image 1000
Processed 18 transformations for image 0
Processed 18 transformations for image 1300
Traceback (most recent call last):
  File "training/Zach_train_DAVE2.py", line 164, in <module>
    main()
  File "training/Zach_train_DAVE2.py", line 95, in main
    batch_images.append(sample['image name '])  # Append image to batch_images list
KeyError: 'image name '
